setup.py

```diff
@@ -18,7 +18,7 @@ def parse_requirements(filename: str):
 if __name__ == "__main__":
     setup(
         name=PACKAGE_NAME,
-        version="0.5.2",
+        version="0.5.4",
```

Comment:
I think this should be 0.6.0 as it is a breaking change

Reply:
If we make patches to the controller for 0.7.4 it can be 0.6.1, 0.6.2, etc...

Reply:
sounds good - agreed